tests/fixtures/pages.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from tests.pages.pageobjects import MainPage
+from tests.fixtures.drivers import driver
+import tests.test_data.endpoints
+from tests import common
+
+
+@pytest.fixture
+def logined_page(driver: driver):
+    driver.get(tests.test_data.endpoints.velosipednye_shiny_url)
+    main_page = MainPage(driver)
+    main_page.comein_link.click()
```

Comment:
not clear naming or typo

Reply:
what?

Reply:
it's about the missing underscore char inside many names, like:
'comein_link'  - it has to be 'come_in_link'
or 'get_url_userid'  - it has to be 'get_url_user_id'
or 'gorest_users_url'  -  it has to be 'go_rest_users_url'

Reply:
refactored as `sign_in_link`